pdsview/pdsview.py

```diff
@@ -209,9 +249,39 @@ def __init__(self, image_set):
 
         self.display_image()
 
+    def display_values(self, pds_view, button, data_x, data_y):
+        "Display the x, y, and pixel value when the mouse is pressed and moved"
+        try:
+            # When clicking inside the image
+            image = pds_view.get_image()
+            x = round(data_x, 0)
+            y = round(data_y, 0)
+            value = image.get_data_xy(x, y)
+            self.x_value.setText('X: %.0f' % (x))
+            self.y_value.setText('Y: %.0f' % (y))
+            if self.image_set.current_image.ndim == 3:
+                # Show different band values for 3 band images
+                R = str(round(value[0], 3))
+                G = str(round(value[1], 3))
+                B = str(round(value[2], 3))
+                self.pixel_value.setText('R: %s G: %s B: %s' % (R, G, B))
+            elif self.image_set.current_image.ndim == 2:
+                # Show single pixel value for 2 band images
```

Comment:
`Change in Comments`:
Please update `2 band` to `single band`. (Because ndim =2 is `2 dimensional`).